eta/core/video.py

```diff
@@ -27,15 +27,109 @@
 import cv2
 import numpy as np
 
-import eta.core.image as im
 from eta.core import utils
+import eta.core.image as im
```

Comment:
This does not follow the convention
`from eta.core import utils as etau` or just `import eta.core.utils at etau` and
`import eta.core.image as etai`

Reply:
This cleanup is coming in the `veh-sim` branch.